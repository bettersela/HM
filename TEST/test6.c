//+= /= *= %= ++ --; for; switch test
int main(){
	int a;
	int b;
	int i;
	for (i = 0; i <= 6; ++i){
		a = 10;
		b = 6;
		switch (i) {
			case 0:
				(a += b);
			case 1:
				(a -= b);
			case 2:
				(a *= b);
			case 3:
				(a /= b);
			case 4:
				(a %= b);
			case 5:
				(++a);	
			case 6:
				(--a);		
		}
		print a;
	}
}
