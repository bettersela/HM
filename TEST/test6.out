24 19 0 5 25 15 1 15 1 15 1 13 0 2 1 0 0 12 15 -1 16 231 13 0 0 1 0 10 12 15 -1 13 0 1 1 0 6 12 15 -1 13 0 2 1 11 0 0 6 17 65 13 0 0 1 9 11 13 0 1 1 11 1 12 15 -1 13 0 2 1 11 0 1 6 17 90 13 0 0 1 9 11 13 0 1 1 11 2 12 15 -1 13 0 2 1 11 0 2 6 17 115 13 0 0 1 9 11 13 0 1 1 11 3 12 15 -1 13 0 2 1 11 0 3 6 17 140 13 0 0 1 9 11 13 0 1 1 11 4 12 15 -1 13 0 2 1 11 0 4 6 17 165 13 0 0 1 9 11 13 0 1 1 11 5 12 15 -1 13 0 2 1 11 0 5 6 17 187 13 0 0 1 9 11 0 1 1 12 15 -1 13 0 2 1 11 0 6 6 17 209 13 0 0 1 9 11 0 1 2 12 15 -1 13 0 0 1 11 22 15 -1 15 0 13 0 2 1 9 11 0 1 1 12 15 -1 13 0 2 1 11 0 6 10 7 8 18 22 15 -3 21 -1